plugins/obs-qsv11/CMakeLists.txt

```diff
@@ -80,6 +80,11 @@ target_sources(
           QSV_Encoder_Internal.h
           bits/linux_defs.h
           bits/windows_defs.h)
+if(ENABLE_HEVC)
+  target_sources(
+    obs-qsv11 PRIVATE obs-qsv265.c QSV_HevcEncoder.h QSV_HevcEncoder_Internal.h
+                      QSV_HevcEncoder.cpp QSV_HevcEncoder_Internal.cpp)
```

Comment:
Were these files copy/pasted from the Intel SDK? Otherwise follow obs file naming, e.g. lower_case_words.cpp.

Reply:
Never mind. I see the other files. Looks like they were provided to OBS from Intel